pkg/storage/minio/minio.go

```diff
@@ -632,3 +633,33 @@ func (c *Client) IsConnectionPossible(ctx context.Context) (bool, error) {
 
 	return true, nil
 }
+
+func (c *Client) PresignDownloadFileFromBucket(ctx context.Context, bucket, bucketFolder, file string, expires time.Duration) (string, error) {
+	if err := c.Connect(); err != nil {
+		return "", err
+	}
+	if bucketFolder != "" {
+		file = strings.Trim(bucketFolder, "/") + "/" + file
+	}
+	c.Log.Debugw("presigning get object from minio", "file", file, "bucket", bucket)
+	url, err := c.minioClient.PresignedPutObject(ctx, bucket, file, expires)
```

Comment:
will this work for cloud agent?

Reply:
will we add cloud support later?

Reply:
Yes, today. Only the Cloud will be supported initially.